the-collab-lab/tcl-79-smart-shopping-list
# Mm feature/update listitem singlelist styling

## Description
- This PR implements the styles defined in the Figma files for the `Home` and `List` pages, along with all accompanying files.
   - _Slightly reduced the layout width for the navbar on desktop view to ensure the `NavBar` is not too spaced out._
 



## Related Issue

Close #44, #47 



## Acceptance Criteria
The acceptance criteria for this PR are specified in [Issue #14](https://github.com/the-collab-lab/tcl-79-smart-shopping-list/issues/14).



## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.
`enhancement`
`fix`

## Updates

### Before



#### DESKTOP
<img src="https://github.com/user-attachments/assets/e02171f9-4433-47df-9623-299fc9341245" alt="Single List dark mode" width="200"/>
<img src="https://github.com/user-attachments/assets/a0a48b3f-5ae0-4b2c-a8a8-b6a100857426" alt="Your lists dark mode" width="200"/>
<img src="https://github.com/user-attachments/assets/d67e109b-46ad-41a4-880e-c3a8e6d40c5d" alt="Your list Light mode" width="200"/>
<img src="https://github.com/user-attachments/assets/995b3151-3109-415c-876c-50e3266b9f6cd" alt="Single List Light mode" width="200"/>
<img src="https://github.com/user-attachments/assets/21ee9920-ba96-408d-9bf3-3d776cfd35f5" alt="Screenshot 2024-10-10 at 15:08:02" width="100"/>
<img src="https://github.com/user-attachments/assets/c17cacaf-1830-4ff2-b031-05e39372f524" alt="Screenshot 2024-10-10 at 15:08:14" width="100"/>

----
----


### After

#### DESKTOP
<img src="https://github.com/user-attachments/assets/334896c6-e432-408c-9764-6eedb9516073" alt="Screenshot 2024-10-10 at 15:10:45" width="200"/>
<img src="https://github.com/user-attachments/assets/2b3ce742-f2d5-4303-9cea-f13ecb09552a" alt="Screenshot 2024-10-10 at 15:10:58" width="200"/>

#### MOBILE 
<img src="https://github.com/user-attachments/assets/1c507b5a-66e5-4b39-b435-44d1d8486097" alt="Screenshot 2024-10-10 at 14:55:03" width="200"/>
<img src="https://github.com/user-attachments/assets/42060b37-cae3-4dc1-9d13-f751d4b68d24" alt="Screenshot 2024-10-10 at 14:55:19" width="200"/>
<img src="https://github.com/user-attachments/assets/e340b387-0de5-4a5f-8d8c-c302e2cb6bdb" alt="Screenshot 2024-10-10 at 14:55:39" width="200"/>
<img src="https://github.com/user-attachments/assets/8dc35534-1099-432e-97e8-633650c20b93" alt="Screenshot 2024-10-10 at 14:55:57" width="200"/>
<img src="https://github.com/user-attachments/assets/80e7f82d-2f39-4a95-98b5-29803707915a" alt="Screenshot 2024-10-10 at 15:03:01" width="100"/>



## Testing Steps / QA Criteria

- Do a `git pull` and `git checkout mm-feature/update-listitem-singlelist-styling`
- On the `Home` and `List` pages you can see the styles implemented. You can toggle between Light mode and Dark mode to test the theme-switching functionality.


----
 Some styles might still need to be updated depending on the teams preference
 

 maybe adding some sort of background behind list to break out the white and black backgrounds.
 - examples (Not sure about the look):

<img src="https://github.com/user-attachments/assets/f96ea053-65db-4747-ab84-725ffe44a983" alt="" width="150"/>
<img src="https://github.com/user-attachments/assets/d73f56d6-2462-4d4c-a8e2-6c6a572455d1" alt="Screenshot 2024-10-10 at 16:11:11" width="150"/>
<img src="https://github.com/user-attachments/assets/3620b5fc-68d5-4491-83bd-eed0bb3de314" alt="Screenshot 2024-10-10 at 16:11:44" width="150"/>
<img src="https://github.com/user-attachments/assets/97502146-49c7-4416-a9d2-8dd6587a8271" alt="Screenshot 2024-10-10 at 16:13:46" width="150"/>
